### PR TITLE
Fixes tables disappearing when combined with grass in someone's hand

### PR DESCRIPTION
**Pull request by Duny-**
Fixes https://github.com/d3athrow/vgstation13/issues/2453

### CODE REVIEW

File: code/game/objects/items/weapons/table_rack_parts.dm [hunk 1]
```diff
@@ -62,7 +62,7 @@
 			Grass.amount -= 1
 		else
 			del(Grass)
-		new /obj/item/weapon/table_parts/wood/poker( src.loc )
+		new /obj/item/weapon/table_parts/wood/poker( user.loc )
```
Comment (ComicIronic) [inline code/game/objects/items/weapons/table_rack_parts.dm]: get_turf(user), please.

Comment (Duny-) [inline code/game/objects/items/weapons/table_rack_parts.dm]: At the one condition you tell me why

Comment (ComicIronic) [inline code/game/objects/items/weapons/table_rack_parts.dm]: I was going to say mechs, but you can't use hands in those. I suppose it's just a contingency for if humans are ever placed in nullspace and allowed to use their hands, but if that doesn't exist, then what you wrote is fine.

**Comment by Duny-** [comment]
Also fixed in https://github.com/d3athrow/vgstation13/pull/2465 with a bunch of other fixes so I'll just close this

**Pull request closed without merging**